cypress/test/campaignTests/AnnouncementHero.cy.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import AnnouncementHero from '../../../components/campaigns/AnnoucementHero';
+
+describe('AnnouncementHero Component', () => {
+  it('should render the component when the date is within the valid range', () => {
+    // Set the current date to May
+    const mockDate = new Date('2023-05-01T00:00:00Z');
+    cy.clock(mockDate.getTime());
+
+    mount(<AnnouncementHero />);
+
+    // Assert that the component is rendered
+    cy.get('.bg-gray-50').should('exist');
+  });
+
+  it('should not render the component when the date is outside the valid range', () => {
+    // Set the current date to June
+    const mockDate = new Date('2023-06-01T00:00:00Z');
+    cy.clock(mockDate.getTime());
+
+    mount(<AnnouncementHero />);
+
+    // Assert that the component is not rendered
+    cy.get('.bg-gray-50').should('not.exist');
```

Comment:
same here as well, using `data-componentName`
[ref](https://github.com/asyncapi/website/pull/1724/files#r1216601422)

Reply:
sure @imabp adding data-cy and I will keep in mind to add data-cy rather than relying on tags and classes

Reply:
Yes @reachaadrika, please replace the implementation this with `data-cy="nameOfTheComponent"`, and this will be helpful when we check complex properties of an element. :)